docs/manual-instrumentation.md

```diff
@@ -0,0 +1,123 @@
+
+### Install the distribution
+
+Install EDOT Python:
+
+```bash
+pip install elastic-opentelemetry
+```
+
+### Install the available instrumentation
+
+EDOT Python does not install any instrumentation package by default, instead it relies on the
+`opentelemetry-bootstrap` command to scan the installed packages and install the available instrumentation.
+The following command will install all the instrumentations available for libraries found installed
+in your environment:
+
+```bash
+opentelemetry-bootstrap --action=install
+```
+
+> [!NOTE]
+> Add this command every time you deploy an updated version of your application (in other words, add it to your container image build process).
+
+<!-- ✅ Start-to-finish operation -->
+## Send data to Elastic
+
+After installing EDOT Python, configure and initialize it to start sending data to Elastic.
+
+<!-- ✅ Provide _minimal_ configuration/setup -->
+### Configure EDOT Python
+
+To configure EDOT Python, at a minimum you'll need your Elastic Observability cloud deployment's OTLP endpoint and
+authorization data to set a few `OTLP_*` environment variables that will be available when running EDOT Python:
+
+* `OTEL_RESOURCE_ATTRIBUTES`: Use this to add a service name that will make it easier to recognize your application when reviewing data sent to Elastic.
+* `OTEL_EXPORTER_OTLP_ENDPOINT`: The full URL of the endpoint where data will be sent.
+* `OTEL_EXPORTER_OTLP_HEADERS`: A comma-separated list of `key=value` pairs that will
+be added to the headers of every request. This is typically used for authentication information.
+
+You can find the values of the endpoint and header variables in Kibana's APM tutorial. In Kibana:
+
+1. Go to **Setup guides**.
+1. Select **Observability**.
+1. Select **Monitor my application performance**.
+1. Scroll down and select the **OpenTelemetry** option.
+1. The appropriate values for `OTEL_EXPORTER_OTLP_ENDPOINT` and `OTEL_EXPORTER_OTLP_HEADERS` are shown there.
+
+Here's an example:
+
+```sh
+export OTEL_RESOURCE_ATTRIBUTES=service.name=<app-name>
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://my-deployment.apm.us-west1.gcp.cloud.es.io
+export OTEL_EXPORTER_OTLP_HEADERS="Authorization=Bearer P....l"
+```
+
+> [!NOTE]
+> Alternatively, you can use an [APM agent key](https://www.elastic.co/guide/en/observability/current/apm-api-key.html) to authorize requests to an Elastic Observability endpoint. APM agent keys are revocable, you can have more than one of them, and you can add or remove them without restarting APM Server.
+>
+> To create and manage APM Agent keys in Kibana:
+>
+> 1. Go to **APM Settings**.
+> 1. Select the **Agent Keys** tab.
+>
+> When using an APM Agent key, the `OTEL_EXPORTER_OTLP_HEADERS` is set using different auth schema (`ApiKey` rather than `Bearer`). For example:
+> ```sh
+> export OTEL_EXPORTER_OTLP_HEADERS="Authorization=ApiKey TkpXUkx...dVZGQQ=="
+> ```
+
+<!-- ✅ Manually instrument the application and start sending data to Elastic -->
```

Comment:
This is the only chapter that is different than get-started.md

Reply:
If the previous content is exactly the same (I haven't checked), wouldn't be better to just point the users to follow the steps X,Y,Z of the other doc before continuing?

Anyway if you believe this content won't need changes very often then it's also ok to have it here even if it's duplicated.

Reply:
@colleenmcginnis what do yo think?

Reply:
In e6bb031a9833495d20ac00888d4bad0f684fd18f I've reworked the page to act as a next step for get-started.

Reply:
Oh gosh. I don't think my comment from earlier this week went through. 🙈 

> Some thoughts (you can take them or leave them 🙃):
> 
> * I'll leave it up to you as the primary maintainer of these docs whether you want to duplicate content. (I probably wouldn't if I were maintaining these docs since we don't have a content reuse strategy for Markdown files intended to be viewed in GitHub repos, and it would be more difficult to maintain [almost] identical content.)
> * If you do want to keep the doc self contained, you could set the right expectations for users by being more specific in how you frame this doc vs. the get started doc. For example:
>     * Rename the `Get started` doc → `Get started with auto instrumentation` (if that's the right language).  
>     * Rename this doc `Manual instrumentation` → `Get started with manual instrumentation`.
>     * Update [this section of the README](https://github.com/elastic/elastic-otel-python/tree/manual-instrumentation-docs?tab=readme-ov-file#read-the-docs) to say something like this:
>       ```md
>       * **Get started**:
>           * [Auto instrumentation](./docs/get-started.md)
>           * [Manual instrumentation](./docs/manual-instrumentation.md)
>       * [Configuration](./docs/configure.md)
>       ```

I think the solution you went with works well, too. My only additional suggestion is to link to the new page from the README.

Reply:
Added link in the README, thanks @colleenmcginnis !